src/scripts.js

```diff
@@ -180,14 +173,16 @@ function checkEmptyFields(username, password) {
 }
 
 function setUser(username, password) {
+  passwordField.value = '';
   checkPassword(password);
   checkUsername(username);
-  usernameField.value = '';
-  passwordField.value = '';
 
-  // this might be why username03 can log in as if username3
   let id = username.slice(8);
-  if (id) {
+  if (id.startsWith(0)) {
+    domUpdates.showLogInError();
+    errorTag.innerText = 'User does not exist, please try again.';
+    throw new Error('User does not exist');
+  } else if (id) {
```

Comment:
Good work finding a solution to this issue! When the error is thrown, are you responding to it with a message on the DOM?

Reply:
Yes, but I'd like to handle it better, it is showing a separate screen with the error for 4 seconds and then going back to the log in screen